modules.d/90mdraid/module-setup.sh

```diff
@@ -98,14 +98,6 @@ install() {
     inst_rules "$moddir/59-persistent-storage-md.rules"
     prepare_udev_rules 59-persistent-storage-md.rules
 
-    # guard against pre-3.0 mdadm versions, that can't handle containers
-    if ! mdadm -Q -e imsm /dev/null > /dev/null 2>&1; then
-        inst_hook pre-trigger 30 "$moddir/md-noimsm.sh"
-    fi
-    if ! mdadm -Q -e ddf /dev/null > /dev/null 2>&1; then
-        inst_hook pre-trigger 30 "$moddir/md-noddf.sh"
```

Comment:
Then you can also remove the `md-noimsm.sh` and `md-noddf.sh` scripts from the module directory.